.github/workflows/brew_build.yml

```diff
@@ -0,0 +1,18 @@
+name: Build aktualizr on macOS with homebrew bottle
+on: 
+  push:
+    branches:
+      - master
+jobs:
+  macOS:
+    name: Build macOS homebrew bottle for aktualizr release page
+    runs-on: macOS-10.15
+    steps:
+    - name: Build Aktualizr bottle for macOS
+      run: |
+        curl -O "https://raw.githubusercontent.com/advancedtelematic/homebrew-otaconnect/master/aktualizr-dev.rb"
+        VERSION=2020.2
```

Comment:
Agree. `VERSION=${GITHUB_REF#refs/tags/}` should do it.

But also, I had to revert https://github.com/advancedtelematic/homebrew-otaconnect/pull/3; you can't just update the recipe to a build that we know is broken and merge it without review.

Reply:
> you can't just update the recipe to a build that we know
Why it's broken? I checked it a few times, e.g. https://github.com/advancedtelematic/aktualizr/runs/443072124?check_suite_focus=true.
Anyway, it was broken before.

Reply:
> Is it possible to automatically select the latest version? Otherwise I suspect it will be easy to forget to update this. If it has to be hardcoded, then we need to add a line to the release process document to update this after the release is finished.

`brew_build.yml` is specifically aimed for debugging purposes, that version string can be removed at all from brew_build.yaml. main.yml is actually used for building the release version and VERSION is not hardcoded there.

Reply:
> Why it's broken?

As I said in the chat earlier today, it doesn't actually run. It also doesn't pass `brew test`, for the same reason. That's a major problem.

In the previous recipe, yes, the build was broken for some versions of OSX. But that doesn't break anything for existing users. Your change means that anyone who upgrades gets a broken binary installed with no way to get their old, working installation back without mucking around with homebrew manually.

But all that is beside the point, because it shouldn't have been merged in the first place--nobody reviewed it.

Reply:
I apologize for merging it without any review, I just thought we don't have any external users for OSX platform and no one really cares about it.
If we have external users for OSX, why then we have to look for additional justification to get mac machines for our gitlab CI/CD?

Reply:
Not, it's fixed, this recipe should work https://github.com/advancedtelematic/homebrew-otaconnect/blob/fix/OTA-4241/fix-aktualizr-brewing/aktualizr.rb as well as this build https://github.com/advancedtelematic/aktualizr/releases/download/2020.2/aktualizr-2020.2.catalina.bottle.tar.gz

Reply:
> I just thought we don't have any external users for OSX platform and no one really cares about it.

We do indeed! Otherwise we probably wouldn't bother with the effort. There are also internal users that care.

>  If we have external users for OSX, why then we have to look for additional justification to get mac machines for our gitlab CI/CD?

Good point.